bgpd/bgp_zebra.c

```diff
@@ -3408,6 +3408,9 @@ static int bgp_zebra_process_srv6_locator_delete(ZAPI_CALLBACK_ARGS)
 	struct in6_addr *tovpn_sid;
 	struct prefix_ipv6 tmp_prefi;
 
+	if (!bgp)
```

Comment:
I see the same exists for bgp_zebra_process_srv6_locator_add() also. But the check against bgp being NULL is after decode. So might be an option to fix there the sequence also?

Reply:
Sure, done.